### DISCUSSION TITLE
Opening videos in the default video player

### DISCUSSION THREAD

**Discussion by ImSreyas** [sxyazi/yazi#1255] [1. Q&A] [answered]
How can i open videos in the default video player, when i press enter on a video file?
my default video player is vlc.
previously the videos are opened in the mpv player. (I don't want this behaviour), now i have removed the mpv package.
now the videos are not opening! 

The expected behaviour is to open the video or selected videos in vlc media player when i click enter. 

i have checked the default video player for both mp4 and mkv files like below :
(I don't know this is the correct way of doing so) 
```
xdg-mime query default video/mp4
```
```
xdg-mime query default video/mp4
```
and both of them outputed: 
`vlc.desktop`

---
distro : fedora 40
wm : gnome
terminal : kitty
shell : fish

**Comment by xfzv** [comment, marked as answer]
If you haven't modified default `yazi.toml`, it's probably still trying to launch `mpv`:

https://github.com/sxyazi/yazi/blob/main/yazi-config/preset/yazi.toml#L51-L52

```toml
play = [
    { run = 'mpv --force-window "$@"', orphan = true, for = "unix" },
]
```

Just use this instead:

```toml
play = [
    { run = 'vlc "$@"', desc = "VLC", orphan = true }
]
```

and ensure that you're using `play` for videos:

```toml
[open]
rules = [
  { mime = "video/*", use = [ "play", "reveal" ] },
]
```

**Reply by ImSreyas** [reply]
Thanks....❤️
Now it works.
